src/main/resources/com/byclosure/jenkins/plugins/gcloud/GCloudBuildWrapper/config.jelly

```diff
@@ -1,5 +1,9 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
-    xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:a="/lib/auth">
-  <a:credentials title="Google Credentials" field="credentialsId" />
+    xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:a="/lib/auth" xmlns:g="/lib/gcloud" xmlns:c="/lib/credentials">
+
+  <g:selectGCloudInstallation field="installation"/>
+  <f:entry title="Google Credentials">
```

Comment:
I used <c:select> here in replacement for <a:credentials> as I'm working on improving the later so it offer the "add" button as <c:select> does